cli/command/service/opts.go

```diff
@@ -457,6 +457,33 @@ func convertExtraHostsToSwarmHosts(extraHosts []string) []string {
 	return hosts
 }
 
+type isolationOpts struct {
+	value  *container.Isolation
+	source string
+}
+
+func (o *isolationOpts) Set(value string) error {
+	o.source = value
+	tv := container.Isolation(value)
+	o.value = &tv
+	return nil
+}
+
+func (o *isolationOpts) Type() string {
+	return "isolation"
```

Comment:
This type is what gets reported in the help text. I wonder if `string` would be more helpful to users.